Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
     "apollo-router-core",
     "apollo-router-benchmarks",
     "examples/async-auth",
-    "examples/cookies-to-headers",
+    "examples/rhai-cookies-to-headers",
```

Comment:
I don't like embedding "rhai" into the name of the directory. It's annoyingly inconsistent then with the Cargo package name and the various file names within the directory. Also: I'm not 100% sure, but I think this directory is referenced within the rhai.mdx file as an example.